app/javascript/components/sponsor_us/SponsorshipOption.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Card from 'components/Card';
+
+const SponsorshipOption = ({ title, amount, selected }) => {
+    return (
+        <Card className={`sponsorship-option ${selected ? 'selected' : null}`}>
+            <div className="flex flex-row justify-between">
+                <div className="sponsorship-option-radio flex flex-row items-center">
+                    <input type="radio" value={title} checked={selected} readOnly />
```

Comment:
I wanted to call out this radio input component -- it seems like the only way to create custom radio button styles is to hide the actual component, and then build a new visual component that displays the styles you want. 

I learned this technique from [this StackOverflow post](https://stackoverflow.com/questions/4253920/how-do-i-change-the-color-of-radio-buttons).

You can see the styles for this component [here](https://github.com/wnbrb/wnb-rb-site/pull/79/files#diff-220a6f207a89ff66be583b72662f2c6a1cfdf659f6a1fc20f7ed880fcccb066eR73).

Because the underlying radio input component still exists on the page and has the correct state associated with it, I think this page will still be accessible and properly read by screen readers.